src/compile/data/aggregate.ts

```diff
@@ -26,18 +26,18 @@ function addDimension(dims: {[field: string]: boolean}, fieldDef: FieldDef<strin
   return dims;
 }
 
-function mergeMeasures(parentMeasures: Dict<Dict<boolean>>, childMeasures: Dict<Dict<boolean>>) {
-  for (const field in childMeasures) {
-    if (childMeasures.hasOwnProperty(field)) {
+function mergeMeasures(parentMeasures: Dict<Dict<string>>, childMeasures: Dict<Dict<string>>) {
+  for (const f in childMeasures) {
+    if (childMeasures.hasOwnProperty(f)) {
       // when we merge a measure, we either have to add an aggregation operator or even a new field
-      const ops = childMeasures[field];
+      const ops = childMeasures[f];
       for (const op in ops) {
         if (ops.hasOwnProperty(op)) {
-          if (field in parentMeasures) {
+          if (f in parentMeasures) {
             // add operator to existing measure field
-            parentMeasures[field][op] = true;
+            parentMeasures[f][op] = field({field: f});
```

Comment:
I guess `ops[op]` is the right value here.

One potential issue here is that if `parentMeasures[f][op] ` already exist, and `ops[op]` is different, the merge will be incorrect.  The right solution is to make measure `Dict<Dict<string>>` -- but that's hairy -- feel free to file a bug  with very descriptive description.